src/create_tridiagonal.cpp

```diff
@@ -4,38 +4,50 @@
 // - lower diagonal: vector a, lenght n-1
 // - main diagonal:  vector d, lenght n
 // - upper diagonal: vector e, lenght n-1
-arma::mat create_tridiagonal(const arma::vec& a, const arma::vec& d, const arma::vec& e)
+arma::mat create_tridiagonal(const arma::vec& a, const arma::vec& d,const arma::vec& e)
 {
-  // Start from identity matrix
-  // arma::mat A = arma::mat(n, n, arma::fill::eye);
-
-  // Fill first row (row index 0)
-
-  // Loop that fills rows 2 to n-1 (row indices 1 to n-2)
-  
-  // Fill last row (row index n-1)
-    arma::mat A;
+    // Start from identity matrix
+    int N = d.n_elem;
```

Comment:
```suggestion
    const int N = d.n_elem;
```
Tror det er en god vane å gjøre alt man ikke skal endre til consts, for å sørge for at man ikke gjør noen logiske feil senere.